packages/orbit-components/src/Itinerary/Itinerary.stories.tsx

```diff
@@ -163,210 +137,492 @@ const content = [
+  render: args => {
+    return (
+      <div className="space-y-800">
+        <Itinerary>
+          <ItinerarySegment
+            {...args}
+            banner={
+              <Stack inline align="stretch">
+                <ItinerarySegmentBanner>
+                  <ItineraryBadgeList>
+                    <ItineraryBadgeListItem icon={<BaggageCheckedNone />}>
+                      You can’t bring checked or cabin baggage.
+                    </ItineraryBadgeListItem>
+                    <ItineraryBadgeListItem icon={<Guarantee />}>
+                      Connection protected by the Kiwi.com Guarantee
+                    </ItineraryBadgeListItem>
+                  </ItineraryBadgeList>
+                </ItinerarySegmentBanner>
+              </Stack>
+            }
+          >
             <ItinerarySegmentStop
               city="Prague"
               station="Václav Havel Airport Prague (PRG)"
               date="Fri, 19.10"
-              type="critical"
               time="14:05"
-              cancelledTime="12:50"
             />
             <ItinerarySegmentDetail duration="2h 30m" summary={<BadgeGroup />} />
-            <ItinerarySegmentStop
-              city="Vienna"
-              type="critical"
-              station="Vienna International Airport"
-              date="Fri, 19.10"
-              time="15:35"
-              cancelledTime="14:00"
-            />
-          </ItinerarySegment>
-        </ItineraryStatus>
-        <ItineraryBadgeList spaceAfter="medium">
-          <ItineraryBadgeListItem icon={<AlertCircle />}>
-            The layover in Vienna is too short
-          </ItineraryBadgeListItem>
-        </ItineraryBadgeList>
-        <ItineraryStatus type="warning" label="Affected connection">
-          <ItinerarySegment noElevation>
-            <ItinerarySegmentStop
-              city="Vienna"
-              station="Vienna International Airport"
-              date="Fri, 19.10"
-              time="18:15"
-            />
-            <ItinerarySegmentDetail duration="2h 30m" summary={<BadgeGroup />} content={content} />
             <ItinerarySegmentStop
               city="Milan"
               station="Milan Bergamo International Airport (BGY)"
               date="Fri, 19.10"
-              time="19:20"
+              time="16:35"
             />
           </ItinerarySegment>
-        </ItineraryStatus>
-      </Itinerary>
-      <Itinerary>
-        <ItineraryStatus type="info" label="Added stopover" spaceAfter="medium">
-          <ItinerarySegment noElevation>
+        </Itinerary>
+      </div>
+    );
+  },
+
+  parameters: {
+    controls: {
+      exclude: ["onClick", "onCollapse", "onExpand", "banner"],
+    },
+  },
+
+  args: {
+    onClick: action("clicked"),
+    onCollapse: action("collapsed"),
+    onExpand: action("expanded"),
+    spaceAfter: SPACINGS_AFTER.MEDIUM,
+    noElevation: false,
+    actionable: true,
+    banner: undefined,
```

Comment:
`banner` prop is static in this example. I didn't add it as a dynamic control as it's React.ReactNode type and it would be problematic for the user to set ReactNode prop in controls. I can remove the `banner` prop at all, but the reason behind this is that users can notice that they can use this prop for an `ItinerarySegment` component as well.

Reply:
but it's true that arg control `banner` can be out of the specified `args` at all. 👍